backend/calc.py

```diff
@@ -85,6 +85,9 @@ def get_gcd(self):
     def get_dot_scalar(self):
         return  1 + (self.speed.get_multiplier() / 1000)
 
+    def calc_piety(self):
+        return 200 + self.pie.get_multiplier()
```

Comment:
Should this be pie.get_value()?  I'm not 100% sure what this method is returning.

Reply:
no this is correct -- get_multiplier does a lot of stupid shit because that's just how the formulas in this game work. they all follow the same form of (x * delta + y) and the get_multiplier function pulls out that math and hides the magic numbers